modules/041-linstor/hooks/metrics.go

```diff
@@ -0,0 +1,302 @@
+/*
+Copyright 2023 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package hooks
+
+import (
+	"fmt"
+
+	"github.com/deckhouse/deckhouse/go_lib/telemetry"
+	"github.com/flant/addon-operator/pkg/module_manager/go_hook"
+	"github.com/flant/addon-operator/sdk"
+	"github.com/flant/shell-operator/pkg/kube_events_manager/types"
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const (
+	linstorNodesSnapshot     = "linstor_nodes"
+	linstorSPsSnapshot       = "linstor_storagepools"
+	linstorRDsSnapshot       = "linstor_resource_definitions"
+	linstorResourcesSnapshot = "linstor_resources"
+	linstorCRDsSnapshot      = "linstor_crds"
+	nodeTypeController       = 1
```

Comment:
@kvaps maybe remove this? Golang linter said it's not used anymore

Reply:
sude, done